_pages/tutorials/recipes/content-aware-resize.md

```diff
@@ -8,6 +8,6 @@ author: jon_peck
 excerpt: "Introducing ContentAwareResize, a microservice which crops images to any size… while ensuring that the important parts do not get removed!"
 
 repository: https://gist.github.com/peckjon/a34d6fd227bbb30ed9df6931c082fdc1
-blog: https://blog.algorithmia.com/smart-autogeneration-of-thumbnails-banners-and-socials/
-thumbnail: https://blog.algorithmia.com/wp-content/uploads/2017/06/content-aware-resize.png
+blog: https://algorithmia.com/blog/smart-autogeneration-of-thumbnails-banners-and-socials/
+thumbnail: https://algorithmia.com/blog/wp-content/uploads/2017/06/content-aware-resize.png
```

Comment:
I didn't realize we have links to assets within `wp-content`. That's a bit of a foot gun. We'll have to keep this in mind when we wrap up the hackathon project otherwise these will all break.